portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> canadianCities;
+
+  @Override
+  public void init(){
+    canadianCities = new ArrayList<>();
+    canadianCities.add("Canmore, Alberta"); 
+    canadianCities.add("St. Johns, Newfoundland"); 
+    canadianCities.add("Kingston, Ontario"); 
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello and welcome to my portfolio!");
+    //response.setContentType("text/html;");
```

Comment:
please don't leave commented out code. You can always find it in history if needed again.